src/iris/api.py

```diff
@@ -3170,6 +3170,57 @@ def on_get(self, req, resp, username):
         resp.body = ujson.dumps(user_data)
 
 
+class ValidTarget(object):
+    allow_read_no_auth = True
```

Comment:
We want to set `allow_read_no_auth` to `False`, to make these two routes only work if you're using an API key or a logged in user.